src/fedora/30/amd64/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM fedora:30
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
+# this does not include libraries that we need to compile different projects, we'd like
+# them in a different layer.
+RUN dnf install -y \
+        clang \
+        cmake \
+        findutils \
+        gdb \
+        lldb-devel \
+        llvm-devel \
+        make \
+        python \
+        python2-lldb \
+        sudo \
+        which \
+    && dnf clean all
+
+# Install tools used by build automation.
+RUN dnf install -y \
+        git \
+        tar \
+        procps \
+        zip \
+    && dnf clean all
+
+# Dependencies of CoreCLR and CoreFX.
+RUN dnf install -y \
+        compat-openssl10-devel \
+        glibc-locale-source \
+        iputils \
+        krb5-devel \
+        libcurl-devel \
+        libgdiplus \
+        libicu-devel \
+        libunwind-devel \
+        libuuid-devel \
+        lttng-ust-devel \
+        uuid-devel \
+    && dnf clean all
+
+# Until official packages are available, we have to restore the ubuntu ones instead.
+ENV __PUBLISH_RID=ubuntu.14.04-x64
```

Comment:
I've never understood the need for this, can you explain?  If this is needed, should it be updated to something closer to Fedora 30 such as Fedora 29?

Reply:
I don't understand it either, I copied it off the Fedora 29 dockerfiles.  Should I delete it?

Reply:
I vote to delete it until someone needs it in which they can add it and set it accordingly.

Reply:
Done